tests/utils/inputparams/testparams.go

```diff
@@ -23,8 +23,17 @@ import (
 	"os"
 	"strconv"
 	"time"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/govc"
```

Comment:
Nope this is not a regression issue. govc library loads ssh library which initializes `sshIdentity` so above proposed fix is required to go in with this PR or with separate PR before this one. Instead of waiting for merge to master and rebase this PR .. I would like to request Govindan to supply a fix. 

@govint  Let me know what do you think? Looking forward to your response.